### PR TITLE
Sheila wang/eslint

### LINKED ISSUE #2: Install and configure ESLint
Completion criteria:
- ESLint is installed
- Any necessary plugins/extensions are installed
- An .eslintrc file exists containing sensible settings (accounting for TypeScript/Node/Express)
- An NPM script exists to run eslint (with --fix)

### PR DESCRIPTION
**Pull request by sheila-wang**
`npm run lint` results in errors in `index.ts`
```
  1:1  error    Prefer default export                                     import/prefer-default-export
  2:3  error    Expected 'this' to be used by class method 'getGreeting'  class-methods-use-this
  8:1  warning  Unexpected console statement                              no-console
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,17 +11,23 @@
...
+    "typescript": "^4.4.4"
   },
   "scripts": {
     "build": "tsc",
     "start": "tsc & node .",
     "dev": "tsc -w & nodemon .",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "format": "prettier -c src/*"
+    "format": "prettier -c src/*",
+    "lint": "npx eslint src/*"
```
Comment (jvcorredor) [inline package.json]: This shouldn't need the npx prefix. That's for when you need run scripts in ./node_modules/bin from the command line.

**Comment by jvcorredor** [comment]
Nice setup overall. Please see 1 minor comment.

**Review by jvcorredor — CHANGES_REQUESTED** [review]
One minor change requested.

File: package.json (hunk 1 above)
Comment (sheila-wang) [reply package.json]: resolved

**Review by jvcorredor — APPROVED** [review]
LGTM

**Review by cfgyer — APPROVED** [review]

**Review by samj01 — APPROVED** [review]

**Pull request merged**